src/libraries/Microsoft.PowerFx.Interpreter/RecalcEngine.cs

```diff
@@ -151,7 +162,12 @@ public void UpdateVariable(string name, FormulaValue value)
             else
             {
                 // New
-                var slot = _symbolTable.AddVariable(name, value.Type, mutable: true);
+                if (newVarProps == null)

```

Comment:
If variable already exists, does that mean we would not honor `newVarProps` props? We may want to document that.